blas/unit_test/Test_Blas1_nrm2w.hpp

```diff
@@ -22,28 +22,32 @@
 namespace Test {
 template <class ViewTypeA, class Device>
 void impl_test_nrm2w(int N) {
-  typedef typename ViewTypeA::value_type ScalarA;
-  typedef Kokkos::ArithTraits<ScalarA> AT;
+  using ScalarA    = typename ViewTypeA::value_type;
+  using AT         = Kokkos::ArithTraits<ScalarA>;
+  using MagnitudeA = typename AT::mag_type;
 
   ViewTypeA a("A", N);
   ViewTypeA w("W", N);
 
   typename ViewTypeA::HostMirror h_a = Kokkos::create_mirror_view(a);
   typename ViewTypeA::HostMirror h_w = Kokkos::create_mirror_view(w);
 
+  const MagnitudeA max_val = 10;
```

Comment:
Consider constexpr for `max_val` and `eps`.

Reply:
I don't disagree although both are perfectly correct, I think it was not even declared `const` previously.
Since the PR failed I'll take this opportunity to do the change.